doc/source/singlevisit.rst

```diff
@@ -0,0 +1,544 @@
+created as a result of single-visit processing.
+
+.. list-table:: Single-visit product filenames
+  :widths: 8 25 83
+  :header-rows: 1
+  
+  * - Product
+    - File Type
+    - Filename for Files Produced
+  * - Exposure
+    - drizzle product
+    - hst_<propid>_<obsetid>_<instr>_<detector>_<filter>_<ipppssoo>_dr[cz].fits
+  * -
+    - flat-field product
+    - hst_<propid>_<obsetid>_<instr>_<detector>_<filter>_<ipppssoo>_fl[ct].fits
+  * - 
+    - headerlet file
+    - hst_<propid>_<obsetid>_<instr>_<detector>_<filter>_<ipppssoo>_hlet.fits
+  * -
+    - trailer file
+    - hst_<propid>_<obsetid>_<instr>_<detector>_<filter>_<ipppssoo>_trl.txt
+  * -
+    - preview (full size)
+    - hst_<propid>_<obsetid>_<instr>_<detector>_<filter>_<ipppssoo>_dr[cz].jpg
+  * - 
+    - preview (thumbnail)
+    - hst_<propid>_<obsetid>_<instr>_<detector>_<filter>_<ipppssoo>_dr[cz]_thumb.jpg
+  * - Filter
+    - drizzle product
+    - hst_<propid>_<obsetid>_<instr>_<detector>_<filter>_<ipppss>_dr[cz].fits
+  * -
+    - point-source catalog
+    - hst_<propid>_<obsetid>_<instr>_<detector>_<filter>_<ipppss>_point-cat.ecsv
+  * -
+    - segment-source catalog
+    - hst_<propid>_<obsetid>_<instr>_<detector>_<filter>_<ipppss>_segment-cat.ecsv
+  * - 
+    - trailer file
+    - hst_<propid>_<obsetid>_<instr>_<detector>_<filter>_<ipppss>_trl.txt
+  * -
+    - preview (full size)
+    - hst_<propid>_<obsetid>_<instr>_<detector>_<filter>_<ipppss>_dr[cz].jpg
+  * -
+    - preview (thumbnail)
+    - hst_<propid>_<obsetid>_<instr>_<detector>_<filter>_<ipppss>_dr[cz]_thumb.jpg
+  * - Total
+    - drizzle product
+    - hst_<propid>_<obsetid>_<instr>_<detector>_total_<ipppss>_dr[cz].fits
+  * -
+    - point-source catalog
+    - hst_<propid>_<obsetid>_<instr>_<detector>_total_<ipppss>_point-cat.ecsv
+  * -
+    - segment-source catalog
+    - hst_<propid>_<obsetid>_<instr>_<detector>_total_<ipppss>_segment-cat.ecsv
+  * - 
+    - trailer file
+    - hst_<propid>_<obsetid>_<instr>_<detector>_total_<ipppss>_trl.txt
+  * -
+    - preview (full size)
+    - hst_<propid>_<obsetid>_<instr>_<detector>_total_<ipppss>_dr[cz].jpg
+  * - 
+    - preview (thumbnail)
+    - hst_<propid>_<obsetid>_<instr>_<detector>_total_<ipppss>_dr[cz]_thumb.jpg
+  * - 
+    - color preview (full size)
+    - hst_<propid>_<obsetid>_<instr>_<detector>_total_<ipppss>_<filters>_dr[cz].jpg
+  * - 
+    - color preview (thumbnail)
+    - hst_<propid>_<obsetid>_<instr>_<detector>_total_<ipppss>_<filters>_dr[cz]_thumb.jpg
+    
+
+Processing the Input Data
+=========================
+SVM processing starts with a list of all the single exposures 
+which were taken as part of a visit.  Any associations which were defined by the
+proposal are ignored, since the visit itself gets treated, in essence, as a new 
+association.  The input files can be specified either using the **poller** file format
+used by the STScI automated processing or a file with a simple list of filenames.
+
```

Comment:
I forget.  Is this a simple list of filenames, one filename per line?  If so, we should indicate this.